ecs_task_run/__init__.py

```diff
@@ -117,7 +122,8 @@ def ecs_run():
         run_update_service(cluster_name=args.cluster,
                            image_name=args.image,
                            task_family=args.task,
-                           service_name=args.service)
+                           service_name=args.service,
+                           execution_role_arn=args.executionrolearn)
```

Comment:
Não está claro pra mim: Esse parâmetro é obrigatório? O que vai acontecer se este parâmetro não for passado?

Reply:
Parametro obrigatório é definido assim: https://github.com/Creditas/ecs-task-run/blob/master/ecs_task_run/__init__.py#L38

Reply:
O default se não informado vai ser `None` que é o mesmo default esperado no metodo: https://github.com/Creditas/ecs-task-run/pull/7/files#diff-7f8f2e396d5a8a08d585d13faaa36f0aR73